### ISSUE TITLE
[a11y] Check that there are not duplicate h1 tags on pages

### ISSUE THREAD

**Issue by mattwr18** [tactilenews/100eyes#1812]
I recently realized I was misusing the Heading component, where I passed in the styles for an h2, h3 tag, but did not specify that the tag itself should be used. Having multiple h1 tags is technically possible, but is not recommended for a11y reasons.

See this blog post: [Should you use multiple %h1 heading elements on your page in 2022?](https://blog.shimin.io/should-you-use-multiple-h1-s-in-2022/)
MDN docs: [Avoid using multiple %h1 elements on one page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements?ref=blog.shimin.io#avoid_using_multiple_h1_elements_on_one_page)
